dotnet/src/SemanticKernel.Skills/Skills.Memory.VectorDB/Qdrant/QdrantMemoryStore.cs

```diff
@@ -0,0 +1,201 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Please ensure that QdrantMemoryStore() can seamlessly replace VolatileMemoryStore() in all of our samples. Example: Example15_MemorySkill.cs

Reply:
Yes this was a part of the initial test. Per our conversation, will retest with updates made to this PR.